daq/runner.py

```diff
@@ -273,7 +273,7 @@ def _handle_system_idle(self):
             LOGGER.debug('No active device ports, waiting for trigger event...')
 
     def _reap_stale_ports(self):
-        for port, port_info in self._port_info.items():
```

Comment:
did this get pulled in from another change?

Reply:
It is unrelated to the tech-support changes, but came up as an integration test failure for this PR. Shall I split it out to a separate PR?

Reply:
That was for fixing the “dict size changed during iteration problem” in aux test.